### ISSUE TITLE
Hello, when I interrupt training and restart training from resume_state, learning_rate will not be updated.

### ISSUE THREAD

**Issue by yanghedada** [xinntao/BasicSR#327]
Hello, when I interrupt training and restart training from resume_state, learning_rate will not be updated.
When I was at the 3000 terminal, I started training at 3000. At 80,000, 12,000 learning rate will not change.
@xinntao 
```
train:
  optim_g:
    type: Adam
    lr: !!float 1e-4
    weight_decay: 0
    betas: [0.9, 0.99]

  scheduler:
    type: MultiStepLR
    milestones: [80000, 120000, 150000]
    gamma: 0.5

  total_iter: 200000
  warmup_iter: -1  # no warm up

  # losses
  pixel_opt:
    type: L1Loss
    loss_weight: 1.0
    reduction: mean
```

**Comment by yanghedada** [comment]
I made the following modification, the learning rate can be changed, I don't know if it is correct.
@xinntao 
@Cugtyt 
```
    def resume_training(self, resume_state):
        """Reload the optimizers and schedulers for resumed training.

        Args:
            resume_state (dict): Resume state.
        """
        resume_optimizers = resume_state['optimizers']
        resume_schedulers = resume_state['schedulers']
        assert len(resume_optimizers) == len(
            self.optimizers), 'Wrong lengths of optimizers'
        assert len(resume_schedulers) == len(
            self.schedulers), 'Wrong lengths of schedulers'
        for i, o in enumerate(resume_optimizers):
            self.optimizers[i].load_state_dict(o)
        # for i, s in enumerate(resume_schedulers):
            # self.schedulers[i].load_state_dict(s)
```

**Comment by xinntao** [comment]
@yanghedada 
 I think we should update the schedulers. 
Your uncomments of these schedulers is not proper. 

I have checked the resume function on my experiments. Their behaviours on learning rate updates are correct. 
 
So I will close this issue first. If you still have issues, just reopen it~
